### PR TITLE
completed css-fonts exercise - Colin Macrae 

### PR DESCRIPTION
**Pull request by colin-macrae**

### CODE REVIEW

File: css-fonts/styles.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+body {
+  font-family: 'Oswald', sans-serif;
```
Comment (shawnkost) [inline css-fonts/styles.css:2]: You should be using 2 separate fonts. One for the headings, and one for the rest of the content

**Review by shawnkost — CHANGES_REQUESTED** [review]
Hey Colin! One change requested below. You should be using 2 separate fonts

File: css-fonts/styles.css (hunk 1 above)
Comment (colin-macrae) [reply css-fonts/styles.css:2]: My bad Shawn!

**Review by JenBowman1 — APPROVED** [review]
👍

**Pull request merged**